tools/c7n_policystream/tests/test_policystream.py

```diff
@@ -187,3 +192,50 @@ def test_cli_stream_basic(self):
             rows[-1]['policy'],
             {'data': {'name': 'lambda-check', 'resource': 'aws.lambda'},
              'file': 'example.yml'})
+
+    def test_stream_remove_file(self):
+        git = self.setup_basic_repo()
+        git.rm('example.yml')
+        git.commit('remove file')
+
+        policy_repo = policystream.PolicyRepo(git.repo_path, git.repo())
+        changes = [c.data() for c in policy_repo.delta_stream(
+            sort=pygit2.GIT_SORT_TOPOLOGICAL | pygit2.GIT_SORT_REVERSE)]
+        self.assertEqual(
+            [(c['change'],
+              c['policy']['data']['name'],
+              c['commit']['message'].strip()) for c in changes],
+            [('add', 'codebuild-check', 'add something'),
+             ('remove', 'codebuild-check', 'switch'),
+             ('add', 'lambda-check', 'switch'),
+             ('remove', 'lambda-check', 'remove file')])
+
+    def test_stream_move_policy(self):
+        git = self.setup_basic_repo()
+        git.change('newfile.yml', {
+            'policies': [{
+                'name': 'ec2-check',
+                'resource': 'aws.ec2'}]})
+        git.commit('new file')
+        git.rm('example.yml')
```

Comment:
per other thread on a test case, deleting a file is a different consideration then simply moving a policy. there isn't a policy file to delta against in that case afaics which is what the logic being removed was doing.

Reply:
i'll play around with it.